docs/recipes.md

```diff
@@ -107,6 +107,44 @@ for path in sorted(Path("src").rglob("*.py")):  # (1)
 8. Actually write to the magic file.
 9. We can even set the `edit_uri` on the pages.
 
+> NOTE:
+> It is important to look out for correct edit page behaviour when using generated pages,
+> as `full_path` will be relative to the location of your `mkdocs.yml` path. You can fix
+> incorrect linking by traversing tree and adding it to the path before passing
+> it to `set_edit_path`.
+>
+> For example, if we have `edit_uri` set to `blob/master/docs/` and the following
+> file structure:
```

Comment:
I find the first paragraph confusing. The second one is better as it shows rather than telling. So I suggest simplifying it like so:

```suggestion
> It is important to look out for correct edit page behaviour when using generated pages.
> For example, if we have `edit_uri` set to `blob/master/docs/` and the following
> file structure:
```